### ISSUE TITLE
Taint issue when right clicking 'Human Orphan Whistle'

### ISSUE THREAD

**Issue by Luke1410** [Auctionator/Auctionator#1501]
## Brief Description of Bug

When right clicking the Human Orphan Whistle (https://www.wowhead.com/item=18598/human-orphan-whistle) I get the following error:
![image](https://github.com/Auctionator/Auctionator/assets/15809174/c1e9ceba-7d98-443d-bc25-ab66b59ab2db)

## Loaded Addons

![image](https://github.com/Auctionator/Auctionator/assets/15809174/4a72570f-f44d-4253-b3ba-9ce043dfe983)
![image](https://github.com/Auctionator/Auctionator/assets/15809174/63eabaa7-445a-44b5-8a14-a8d6d5189cce)

## Back Trace (LUA Error Output)

Here's the taint log excempt:
5/1 17:00:57.688  Interface/FrameXML/ContainerFrame.lua:1608 GetBagID()
5/1 17:00:57.688  An action was blocked because of taint from Auctionator - 
5/1 17:00:57.688      UseContainerItem()
5/1 17:00:57.688      Interface/FrameXML/ContainerFrame.lua:1428
5/1 17:00:57.688      ContainerFrameItemButton_OnClick()
5/1 17:00:57.688      Interface/FrameXML/ContainerFrame.lua:1480

Blind guess: Maybe same issue which led TitanPanel to temporarily disable the open bag functionality in their bag plugin?

**Comment by plusmouse** [comment]
Did this happen while at the Auctionator, or elsewhere?

**Comment by Luke1410** [comment]
> Did this happen while at the auction house, or elsewhere?

No, happened while standing in Stormwind next to the Fishing Trainer offering the Daily Fishing quest.

**Comment by Luke1410** [comment]
Not sure about how exactly this can be triggered, but seems to require visiting the AH first (and maybe doing a full scan and/or posting auctions).

**Comment by plusmouse** [comment]
Absent further reports going to view this taint issue as a fluke.
